docs/source/_json/users.json

```diff
@@ -0,0 +1,26 @@
+GET /plone/@users
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+[
```

Comment:
@tisto I see, no worries ;-) Are you planning on merging the basic implementation for `1.0a1` already? Because I feel we've got a pretty stable state in `master` right now, and that's what I recently pushed to production. So I'd love it if we could lock in that state by doing a first release, and then track further changes with a proper change log.